blog/_posts/2019-09-12-tips-on-moving-your-dev-env-from-docker-compose-to-kubernetes.md

```diff
@@ -0,0 +1,173 @@
+---
+slug: tips-on-moving-your-dev-env-from-docker-compose-to-kubernetes
+date: 2019-09-12
+author: nick
+layout: blog
+title: "Tips on Moving your Dev Env from Docker-Compose to Kubernetes"
```

Comment:
Nit (throughout): `docker-compose` is the command line invocation, `Docker Compose` is the tool name

Reply:
good point! updated